packages/ui/src/forum/types/ForumThread.ts

```diff
@@ -11,6 +10,7 @@ export interface ForumThread {
   categoryId: string
   tags: ForumThreadTag[]
   visiblePostsCount: number
+  status: string
```

Comment:
I mean for this `status` field

Reply:
You're absolutely right, I think I put it in as a string for the meantime at first and then just forgot